src/components/LikeButton/LikeButton.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import styles from './LikeButton.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { ReactComponent as Heart } from '../../icons/raw/Heart.svg';
+import { IconToggle, Placement, OnChangeEvent } from '../IconToggle';
+
+export interface LikeButtonProps {
+  label: React.ReactNode | string;
+  labelPlacement: Placement;
+  checked: boolean;
+  onChange?: (event: OnChangeEvent) => void;
+  disabled: boolean;
+}
+
+interface DefaultProps {
+  label: number;
+  labelPlacement: Placement;
+  checked: boolean;
+  disabled: boolean;
+}
+
+/** like button */
+export class LikeButton extends React.Component<LikeButtonProps> {
+  static displayName = 'LikeButton';
+  static defaultProps: DefaultProps = { 
+    label: 0,
+    labelPlacement: 'right',
+    checked: false,
+    disabled: false,
+  };
+
+  render() {
+    const { label, labelPlacement, onChange, checked, disabled, ...rest } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile }) => (
+            <div className={styles.likeButton}>
```

Comment:
I think this wrapping `div` can also be removed, no?

Reply:
Both TPAComponentsConsumer and div were removed.